packages/venia-ui/lib/components/AddressBookPage/addressBookPage.js

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo } from 'react';
+import { PlusSquare } from 'react-feather';
+
+import { useAddressBookPage } from '@magento/peregrine/lib/talons/AddressBookPage/useAddressBookPage';
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+
+import { GET_CUSTOMER_ADDRESSES } from '../CheckoutPage/AddressBook/addressBook.gql';
```

Comment:
Prefer to use new gql rather than re-use even if they happen to currently use the same info. In the future we may change address book in checkout to require different info which would break this page.